src/BookNavigator/book-navigator.js

```diff
@@ -2,7 +2,7 @@
 import { SharedResizeObserver } from '@internetarchive/shared-resize-observer';
 // eslint-disable-next-line no-unused-vars
 import { ModalManager } from '@internetarchive/modal-manager';
-import { css, html, LitElement } from 'lit-element';
+import { css, html, LitElement } from 'lit';
 import { nothing } from 'lit-html';
```

Comment:
`lit-html` no longer exists! It looks like it's in `'lit'` now, so add it to the above section. There should be no occurrences of `lit-html` in the codebase.